packages/shared/src/contexts/SettingsContext.tsx

```diff
@@ -53,6 +54,8 @@ export interface SettingsContextData extends Omit<RemoteSettings, 'theme'> {
   updateCustomLinks: (links: string[]) => Promise<unknown>;
   syncSettings: (bootUserId?: string) => Promise<unknown>;
   onToggleHeaderPlacement(): Promise<unknown>;
+  setOnboardingChecklistView: (value: ChecklistViewState) => Promise<unknown>;
+  setSettings: (newSettings: Partial<RemoteSettings>) => Promise<void>;
```

Comment:
Exposed it so I can set settings during onboarding to default values (checklist and sidebar state) in one go.